### DISCUSSION TITLE
Local Library?

### DISCUSSION THREAD

**Discussion by kdjanz** [ciderapp/Cider#729] [Q&A] [answered]
I have not signed up for Apple Music, but have a server full of music that I have imported to iTunes/Music on my Mac. I have extensive ratings and smart playlists built around those. 

How do I make my local Library appear in Cider?

Thanks

Kelly

**Comment by quacksire** [comment, marked as answer]
We don't support local files/non Apple Music files (imported to iTunes Match/Sync is fine, if you have AM) but this is certainly a thing we are looking into

**Comment by TheStakman** [comment]
I'm in the same boat, I found this when trying to see if there was a way to give iTunes a dark mode and got all excited to switch over to Cider. Only to be heartbroken when I realized I had to sign up for Apple music... I just want to listen to my local library of music in a fresh dark music player. Please add this feature, I'll gladly pay $5-$20 for Cider if this is added.

**Reply by AW1534** [reply]
you'd pay $20 for dark mode?
